pkgs/games/wargus/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, callPackage, fetchFromGitHub
+, cmake, pkg-config, makeWrapper
+, zlib, bzip2, libpng
+, dialog, python, cdparanoia
```

Comment:
Does this need to be python 2 or 3? I think we should set this explicit just to make it clear.